mono/metadata/w32file-win32.c

```diff
@@ -393,14 +393,18 @@ mono_w32file_move (const gunichar2 *path, const gunichar2 *dest, gint32 *error)
 	return result;
 }
 
+#endif
+
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT) || G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT)
```

Comment:
I think, you would prefer, this identical content be in icall-windows-uwp.c, instead of icall-windows.c? Identical content and identical if, still used by both classic and uwp?

Reply:
And a few others ditto.